File: app/src/main/java/gay/nihil/lena/heartbeat/AppDatabase.java
package gay.nihil.lena.heartbeat;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {User.class, Event.class}, version = 2)
public abstract class AppDatabase extends RoomDatabase {

    public abstract UserDao userDao();

    public abstract EventDao eventDao();
}

File: app/src/main/java/gay/nihil/lena/heartbeat/AppDatabase.kt
package gay.nihil.lena.heartbeat

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(entities = [User::class, Event::class], version = 2)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): UserDao?

    abstract fun eventDao(): EventDao?
}